modules/Timetable Admin/course_rollover.php

```diff
@@ -193,17 +193,15 @@
             echo __('The next school year cannot be determined, so this action cannot be performed.');
             echo '</div>';
         } else {
-            try {
-                $dataNext = array('gibbonSchoolYearID' => $nextYear);
-                $sqlNext = 'SELECT * FROM gibbonSchoolYear WHERE gibbonSchoolYearID=:gibbonSchoolYearID';
-                $resultNext = $connection2->prepare($sqlNext);
-                $resultNext->execute($dataNext);
-            } catch (PDOException $e) {
-                echo "<div class='error'>".$e->getMessage().'</div>';
-            }
-            if ($resultNext->rowCount() == 1) {
-                $rowNext = $resultNext->fetch();
-            }
+            
+            $dataCurrent = array('gibbonSchoolYearID' => $_SESSION[$guid]['gibbonSchoolYearID']);
+            $sqlCurrent = 'SELECT * FROM gibbonSchoolYear WHERE gibbonSchoolYearID=:gibbonSchoolYearID';
+            $rowCurrent = $pdo->selectOne($sqlCurrent, $dataCurrent);
```

Comment:
Does this not have a deleted section somewhere about? I can see where $dataNext is deleted from, but not $dataCurrent?

Reply:
No deleted section for this one. This was added to grab the school year info to use the lastDay value, which as it happens I removed because setting all the classes to unenrolled first didn't seem useful in the end. I'll tidy it u and remove this section.